cascade-merge/action.yml

```
@@ -0,0 +1,198 @@
+# more info on https://ledgerhq.atlassian.net/l/cp/Ar8F1nmM
+name: 'Auto Cascade Merge'
+description: 'Automatically cascade merge changes from one branch to others.'
+branding:
+  icon: 'git-pull-request'
+  color: 'purple'
+
+inputs:
+  gh-token:
+    description: 'pass GITHUB_TOKEN'
+    required: true
+  default-branch:
+    description: 'Default branch name (e.g., main or master)'
+    required: true
+    default: 'main'
+  slack-webhook-url:
+    description: 'SLACK_WEBHOOK_URL'
+    required: false
+  slack-api-token:
+    description: 'SLACK_API_TOKEN'
+    required: false
+  create-pr:
+    description: 'create PR '
+    required: true
+    default: 'true'
```

I think it make more sense to have opt-in "no PR" instead of out-out "do PR"

```suggestion
  auto-merge:
    description: 'Automatically merge branch instead of creating PR '
    required: true
    default: 'false'
```

---

applied